fiftyone/server/view.py

```diff
@@ -571,47 +573,32 @@ def _make_range_query(path: str, field: fof.Field, args):
         mn, mx = None, None
 
     exclude = args.get("exclude", False)
-
-    min_expr = {path: {"$gte": mn}} if mn is not None else None
-    max_expr = {path: {"$lte": mx}} if mx is not None else None
-    if not exclude and range_:
-        return {
-            "$and": [
-                e
-                for e in [
-                    min_expr,
-                    max_expr,
-                ]
-                if e
-            ]
-        }
-
-    min_expr = {path: {"$lt": mn}} if mn is not None else None
-    max_expr = {path: {"$gt": mx}} if mx is not None else None
+    ops = {}
+    if mn is not None:
+        ops["$gte"] = mn
+    if mx is not None:
+        ops["$lte"] = mx
     if range_:
+        if exclude:
+            return {
+                path: {"$not": ops},
+            }
```

Comment:
I agree with the rabbit. The simplifications look good, but tests should be added that assert range handling (include and exclude)

Reply:
I added some tests for single include/exclude as well as multi include/exclude combos.